packages/contracts/package.json

```diff
@@ -18,12 +18,15 @@
   "scripts": {
     "docs": "typedoc && rimraf docs/README.md",
     "prebuild": "rimraf lib",
-    "build": "tsc",
+    "build": "yarn && ./node_modules/.bin/tsc",
```

Comment:
Sorry for the late comment, but i just (re-)saw this. With the new step 7 in the readme, `tsc` command is enough.